### PR TITLE
fixed fixed_string expression eval for go

### PR DESCRIPTION
**Pull request by bright-starry-sky**
fixed bug for expression eval when prop data type is fixed_string
Repetition steps：
```
CREATE SPACE test_space (partition_num=1,replica_factor=1, vid_type=fixed_string(30));
USE test_space
create tag t1(c1 fixed_string(40))
create edge e1(c1 fixed_string(40))
insert vertex t1(c1) values "1":("row"), "2":("row"), "3":("row")
insert edge e1(c1) values "1" -> "2"@0:("row"), "1" -> "3"@0:("row")
go from "1" over e1 where e1.c1 == "row"
go from "1" over e1 where $$.t1.c1 == "row"
go from "1" over e1 where $^.t1.c1 == "row"
```

### CODE REVIEW

File: src/storage/exec/QueryUtils.h [hunk 1]
```diff
@@ -71,6 +71,10 @@ class QueryUtils final {
       }
       return Status::Error(folly::stringPrintf("Fail to read prop %s ", propName.c_str()));
     }
+    if (field->type() == meta::cpp2::PropertyType::FIXED_STRING) {
```
Comment (Shylock-Hg) [inline src/storage/exec/QueryUtils.h:74]: I think you should remove the extra zero in the RowReader.

Comment (bright-starry-sky) [reply src/storage/exec/QueryUtils.h:74]: > I think you should remove the extra zero in the RowReader.

No, there are other effects for reader. such as upgrade tool.

Comment (Shylock-Hg) [reply src/storage/exec/QueryUtils.h:74]: Why upgrade tool need the raw value.

**Review by yixinglu — APPROVED** [review]

**Review by critical27 — APPROVED** [review]
LGTM

**Pull request merged**